lib/elastomer/client/index.rb

```diff
@@ -261,17 +261,17 @@ def flush( params = {} )
         response.body
       end
 
-      # Optimize one or more indices. Optimizing an index allows for faster
-      # search operations but can be resource intensive.
+      # Force merge one or more indices. Force merging an index allows to
+      # reduce the number of segments but can be resource intensive.
       #
       # params - Parameters Hash
-      #   :index - set to "_all" to optimize all indices
+      #   :index - set to "_all" to force merge all indices
       #
-      # See https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-optimize.html
+      # See https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-forcemerge.html
       #
       # Returns the response body as a Hash
-      def optimize( params = {} )
-        response = client.post "{/index}/_optimize", update_params(params, :action => "index.optimize")
+      def forcemerge( params = {} )
+        response = client.post "{/index}/_forcemerge", update_params(params, :action => "index.forcemerge")
         response.body
```

Comment:
Could you please also add an `:optimize` alias for this `:forcemerge` function such that we can maintain backwards compatibility? Please add a "DEPRECATED:" comment above the alias so we know to remove it in the future.

```ruby
# DEPRECATED:  ES 5.X has removed the `/_optimize` endpoint.
alias :optimize :forcemerge
```

Reply:
@TwP sure, just one more thing related to this that I added in the  PR  description. If we just add an alias, I believe this will fail with ES 2.0.x because the API renaming happened in 2.1.

Reply:
I'm fine with breaking ES 2.0. That version of Elasticsearch has been End-of-Lifed by Elastic, so I'm not interested in maintaining support there.

And yes, we are loosey goosey with our version support, and tightening up on version is another project. With parameter validation coming in ES 6.X we will have to be more strict anyways.

Reply:
Understood, thanks for the clarification! :)

Reply:
That's now changed as discussed. Also, I try to exercise the alias in the test.